paths.pegjs

```diff
@@ -45,33 +45,54 @@ ImpliedParenExpr = head:WordOrParen tail:("ᐉ" WordOrParen)+ {
 
 Exists = "$exists"
 
-RemoteLink = "${" remoteLink:PathList "}" {
+RemoteLink = "${" _ remoteLink:PathList _ "}" {
   return { remoteLink }
 }
 
 
 
-MathExpression = operation:("add"/"sub"/"mul"/"div") "(" path:EquationPartList ")" {
+MathExpression = operation:("add"/"sub"/"mul"/"div") _ "(" _ path:EquationPartList _ ")" {
   return {
     operation,
     math: path
   }
 }
 
-CqExpression = operation:("cq") "(" path:Expression ")" {
+LiteralDate = "date" _ "(" path:StringLiteral ")" {
```

Comment:
might be worth allowing optional formatting as second argument (which is the optional second parameter in the `moment` ) https://momentjs.com/docs/#/parsing/string-format/

Reply:
oh, interesting.  that wouldn't be hard to get in.